scripts/breaking_changes_checker/breaking_changes_tracker.py

```diff
@@ -35,6 +35,9 @@ class ChangeType(str, Enum):
     ADDED_CLIENT_METHOD = "AddedClientMethod"
     ADDED_CLASS = "AddedClass"
     ADDED_CLASS_METHOD = "AddedClassMethod"
+    ADDED_CLASS_METHOD_PROPERTY = "AddedClassMethodProperty"
+    ADDED_CLASS_PROPERTY = "AddedClassProperty"
+    ADDED_FUNCTION_PARAMETER = "AddedFunctionParameter"
```

Comment:
New positional params are breaking so those are reported as part of breaking changes. We could add optional in there to make it clearer. I'll open an issue to do that when I circle back to the function parameter checks since they're not complete yet and still need to be tested/verified.

Reply:
https://github.com/Azure/azure-sdk-for-python/issues/36145

Reply:
I don't like the word Optional since it was different meaning in Python 😆. Also, by Optional are you referring to keyword args? I think we'd need to differentiate between positional params, positional-only, keyword-only, and keyword-only required, etc

Reply:
Will make a note of that in the issue to make sure we consider it :)